rules/mandatory-object-comma.js

```diff
@@ -0,0 +1,117 @@
+const { Rule } = require('../api');
+
+module.exports = class extends Rule {
+    getName() {
+        return 'mandatory-object-comma';
+    }
+
+    test(data) {
+        const { formatMessage, addError } = this.outcome;
+
+        this.filterModulesData(data, 'twig').forEach(twigFileData => {
+            const {files, type, name} = twigFileData;
+            const {twig} = files;
+            const {content, path} = twig;
+            const betweenBlocksReg = /^\}[\s]{0,}\{$/i;
+            const twigVariableReg = /\{\{[a-zA-Z0-9 _\=\.\"\%\':\/\_\-\~\,\(\)\?\|\[\]\n\r]{1,}\}\}/gi;
```

Comment:
some variables with RegExp end on RegExp other on Reg. Please change to the common variant

Reply:
I renamed vvariables to RegExp